src/YotiCallbackHandler.ts

```diff
@@ -50,7 +50,13 @@ class YotiCallbackHandler implements LambdaInterface {
 					}
 				}
 				
-				await YotiCallbackProcessor.getInstance(logger, metrics, YOTI_PRIVATE_KEY).processRequest(body);
+				if (body.topic === "session_completion") {
+					await YotiCallbackProcessor.getInstance(logger, metrics, YOTI_PRIVATE_KEY).processRequest(body);
+				} else {
+					logger.warn("Unexpected topic received in request");
+					return new Response(HttpCodesEnum.BAD_REQUEST, "Unexpected topic received in request");
```

Comment:
Ideally we should change the return type for this lambda as its handling SQS messages, it should return SQSBatchResponse type just we have done in PostEvent and GovNotify lambdas.

Reply:
The lambda is built similar to GovNotify whereby it will only retry and fall into DLQ if the **shouldThrow** flag is set when throwing the error: like [here](https://github.com/alphagov/di-ipv-cri-f2f-api/blob/main/src/services/YotiCallbackProcessor.ts#L72)
So in this case it'll return the error but won't go back to the queue to retry